### ISSUE TITLE
Cannot get Virtualbox to work as in the documents

### ISSUE THREAD

**Issue by jcoonrod** [dahliaOS/buildroot#16]
**Describe the bug**
libvert daemon not running

**What kind of a bug is it?**
Trying to run my new iso in virtualbox

**To Reproduce**
1. Issue the installation command in the documentation and run Virtual Machine Manager
2. Click on 'Create New virtual Machine
3. See error: No active connection to install on

**Expected behavior**
Expected to be able to create a new virtual machine

**Screenshots**
[Screenshot from 2022-02-13 16-18-27](https://user-images.githubusercontent.com/54367582/153775470-86ca0ea4-f882-43cc-aadc-5a11620b7551.png)
**Desktop (please complete the following information):**
 - OS: [Ubuntu 20.04.3LS]
 - Device: [ASUS pn41]

**Additional context**
!

**Comment by jcoonrod** [comment]
Found the qemu from cli - that works as far as the login - but what do I do about the password
![Screenshot from 2022-02-13 16-34-21](https://user-images.githubusercontent.com/54367582/153776192-d42da4e0-b516-4161-b5ed-65cef32dc623.png)
?

**Comment by nmcain** [comment]
The password for root is test0000

**Comment by jcoonrod** [comment]
ok - that worked - now not sure how to launch pangolin inside qemu...

**Comment by nmcain** [comment]
Depending on whether or not you copied the overlays to it, you should be able just to type startx

**Comment by jcoonrod** [comment]
Yeah - that's what I thought - when I did that it complains that file ./root/.serverauth.407 does not exist, and later... Fatal server error: (EE) no screens found. So - I guess I didn't "copy overlays to it" (whatever that means). Is there any documentation on that?

**Comment by nmcain** [comment]
I probably should assemble a doc on building from scratch. You just need to `cp -r dahliaos-overlays/* output/target/*` and then rebuild.

**Comment by jcoonrod** [comment]
did the rebuild - doubled the size of the iso - but same errors when logging in and running startx

**Comment by nmcain** [comment]
Try running `source ~/.profile`, if that doesn't work, do `rm /etc/X11/xorg.conf&&startx`

**Comment by jcoonrod** [comment]
Thanks! The second approach worked - it apparently lacks a .profile file.  It launched three terminals but nothing that looks like pangolin desktop - is that a separate command?
![Screenshot from 2022-02-14 17-03-32](https://user-images.githubusercontent.com/54367582/153954448-7848fccf-f219-47db-b6a7-3dd3add9da0a.png)

**Comment by nmcain** [comment]
Make sure that the file /etc/X11/xinit/xinitrc is the one from the overlays, not the stock XOrg config.
